ads/zucks.js

```diff
@@ -42,7 +41,9 @@ export function zucks(global, data) {
       });
     });
   } else if (data['adtype'] === 'native') {
-    loadScript(global, `https://j.zucks.net.zimg.jp/n?f=${data['frameId']}`);
+    const s = global.document.createElement('script');
+    s.src = `https://j.zucks.net.zimg.jp/n?f=${data['frameId']}`;
+    global.document.getElementById('c').appendChild(s);
```

Comment:
cc @zhouyx is this a valid use case to not use `loadScript`? 🤔

Reply:
Hmm I do find `loadScript` can be used here. Can we switch to that helper method? Thanks.

Reply:
How do I access my script dom to appendChild into document.getElementById('c') if I use the loadScript since loadScript does not return anything

Reply:
Interesting, can I know the reason why we must insert the script to the container element? Thanks.

Reply:
https://github.com/ampproject/amphtml/tree/master/ads#the-iframe-sandbox

> Note that the container div (with id="c") is absolute positioned and takes the whole space of the iframe, so you will want to append your content as a child of the container (don't append to body).

without doing this our ad can't be clicked

Reply:
Thanks @yang-wei 

I understand there's this container div, however the container div is within the same window, which means the script will be running with the same context no matter where it is inserted. 

That said I don't think inserting `<script>` to document would break your use case, and I'd be very curious if it does. 

However I you do insist on leaving it as it is, I'm also ok with it. Let me know Thanks.

Reply:
http://zucksdevelopers.vg/amp.html 

Can you look at this ?

![image](https://user-images.githubusercontent.com/5494874/62269527-51020580-b46e-11e9-8215-50c00f1645e8.png)


You can we see add an ungly css
```
body > div#c {
    z-index: -1;
}
```

If you remove this then this ad will become unclickable

Reply:
Thanks @yang-wei , unfortunately the two examples you provide, one has `adType = zoe`, the other has `adType` not defined. I couldn't test with `adType=native`. 

That said, I found you are using the `writeScript` for the other ad type. It's possible that you need the script to be loaded synchronously. If so, please try using `writeScript` method instead. Let me know Thanks

Reply:
you mean use `loadScript` instead right.

I will add another example to show this